permissions/plugin-elastic-axis.yml

```
@@ -4,4 +4,5 @@ github: "jenkinsci/elastic-axis-plugin"
 paths:
 - "org/jenkins-ci/plugins/elastic-axis"
 developers:
-- "taksan"
\ No newline at end of file
+- "taksan"
```

```suggestion
- "takeuchi"
```

AFAICT the user ID is obsolete. CC @taksan

---

Interesting that there are 4 repositories that mention `taksan` and only one that mentions `takeuchi`.

## taksan

* permissions/plugin-autocomplete-parameter.yml:- "taksan"
* permissions/plugin-categorized-view.yml:- "taksan"
* permissions/plugin-chosen-views-tabbar.yml:- "taksan"
* permissions/plugin-elastic-axis.yml:- "taksan"

## takeuchi

* permissions/plugin-categorized-view.yml:- "takeuchi"